Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -113,6 +113,7 @@ public PrintHelper(Panel canvasContainer, PrintHelperOptions defaultPrintHelperO
             _printCanvas.Opacity = 0;
 
             _canvasContainer = canvasContainer;
+            _canvasContainer.RequestedTheme = ElementTheme.Light;
```

Comment:
We should cache the original value in case it was modified by the developer.

Reply:
I always set to App Default (`_canvasContainer.RequestedTheme = ElementTheme.Default;`). So it will always be to what dev had set before.

Reply:
But if they had explicitly set their Panel to be `Dark` theme, the `Default` would be different than that.

Reply:
@avknaidu I'm saying something like this:

```suggestion
            _originalTheme = _canvasContainer.RequestedTheme;
            _canvasContainer.RequestedTheme = ElementTheme.Light;
```

Then we can use `_originalTheme` for resetting the property later when the task is completed.